tests/storage/datavolume.go

```diff
@@ -1196,10 +1229,14 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			dataVolume = dvChange(dataVolume)
 			preallocated := dataVolume.Spec.Preallocation != nil && *dataVolume.Spec.Preallocation
 
-			vmi := tests.NewRandomVMIWithDataVolume(dataVolume.Name)
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512M")
+			vmi := libvmi.New(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithDataVolume("disk0", dataVolume.Name),
+				libvmi.WithResourceMemory("512M"),
+				libvmi.WithCloudInitNoCloudEncodedUserData("#!/bin/bash\n echo hello\n"),
```

Comment:
Same as above

Reply:
Same, not used here?

Reply:
Aaaaaaah that space between `\n` and `echo` :)
Fixed! Thank you